README.md

```diff
@@ -24,6 +24,24 @@ Run
 
 This local development includes hot reloading on the front-end and the back-end.
 
+Make sure that your have the necessary environment varibales available in the `.env` file and the `docker-compose.yml` file
+
+```
+GRYNOS_COURSES_UPDATE_INTERVAL=3600000
```

Comment:
you dont have to list these out. They are in the template files already

Reply:
Yes, I will update.